packages/components/src/components/FunctionChart.tsx

```diff
@@ -58,7 +152,15 @@ export const FunctionChart: React.FC<FunctionChartProps> = ({
     setMouseOverlay(NaN);
   }
   const signalListeners = { mousemove: handleHover, mouseout: handleOut };
-  let mouseItem = runForeign(fn, [mouseOverlay], environment);
+  let mouseItem: result<squiggleExpression, errorValue> = !!mouseOverlay
+    ? runForeign(fn, [mouseOverlay], {
+        sampleCount: 10000,
```

Comment:
Any reason why you are not using the environment handed down? Or if not, use the defaultEnvironment variable exported from @quri/squiggle-lang?

Reply:
Yea, this was an error